config/private/plugins.js

```diff
@@ -57,6 +57,9 @@ export default {
         ].filter(x => x != null),
 
         plugins: [
+          // this is used by mobx, and has to list at first
+          // https://mobxjs.github.io/mobx/best/decorators.html
+          'transform-decorators-legacy',
```

Comment:
Argh, this being said I have noticed we have a stage 2 plugin currently active: "class properties"  
This is a bug.  I am going to set an issue to remove it.